src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -41,15 +49,37 @@ const COLUMN_DATA: Column[] = [
 
 const AuditLogsTable = ({ showModel = false }: Props) => {
   const { modelName } = useParams<EntityDetailsRoute>();
-  const additionalEmptyMsg = showModel ? ` for ${modelName}` : "";
+  const [auditLogs, setAuditLogs] = useState<AuditEvent[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
There is a small flicker (showing table with no data and then the `LoadingSpinner`) before the first `useEffect` is triggered and makes `loading = true`. Maybe we could make `true` to be the default state of `loading`?

Reply:
Done!